test/e2e/testing-manifests/storage-csi/update-hostpath.sh

```diff
@@ -89,19 +89,20 @@ EOF
 
 download () {
     project="$1"
-    path="$2"
-    tag="$3"
-    rbac="$4"
+    localpath="$2"
+    remotepath="$3"
```

Comment:
Can you add comments on what is the localpath vs remotepath?  Why do we need to have both?